src/test/java/com/softserveinc/ita/HomePageTest.java

```diff
@@ -16,7 +17,53 @@ public void verifyLanguageSwitchingTest() {
         var isLanguageSwitched = homePage.isLanguageSwitchedTo(languageToSwitch);
 
         assertThat(isLanguageSwitched)
-                .as("Language should be switched")
+                .as("Language should be switched to '" + languageToSwitch + "'")
                 .isTrue();
     }
+
+    @Test
+    public void verifyCheckoutPageFunctionalityTest() {
+        homePage.closeAdBanner();
+        var lastProductOfSection = homePage.getLastProductOfSection("Акційні пропозиції");
+
+        assertThat(lastProductOfSection)
+                .as("Last product of section should be opened")
+                .isEqualTo(productPage.getProductName());
```

Comment:
what are you checking here? in assert should be verified something from page with some expected value

Reply:
method getLastProductOfSection (variable lastProductOfSection) clicks on last product and returns its name. Im verifying that this name equals to product's name we opened